integration-tests/bats/sql-privs.bats

```diff
@@ -53,6 +53,38 @@ teardown() {
     teardown_common
 }
 
+
+@test "sql-privs: can read json privilege files and convert them" {
+    make_test_repo
+    cp $BATS_TEST_DIRNAME/privs.json .
+
+    # Test that privs.json file is in json format
+    run cat privs.json
+    [[ "$output" =~ "\"User\":\"privs_user\"" ]] || false
+
+    start_sql_server_with_args --host 0.0.0.0 --user=dolt --privilege-file=privs.json
+
+    server_query test_db 1 "select user from mysql.user order by user" "User\ndolt\nprivs_user"
+
+    server_query test_db 1 "create user new_user" ""
+
+    server_query test_db 1 "select user from mysql.user order by user" "User\ndolt\nnew_user\nprivs_user"
+
+    # Test that privs.json file is not in json format
+    run cat privs.json
+    ! [[ "$output" =~ "\"User\":\"privs_user\"" ]] || false
```

Comment:
i think you should stop and restart the sql server after this with the same parameters and make sure it loads the newly saved information permissions